### DISCUSSION TITLE
UTM SE: startup.nsh

### DISCUSSION THREAD

**Discussion by matermel0n** [utmapp/UTM#6599] [Windows]
UTM SE windows XP, just does not work.
every time i try to boot it, **it says press ESC in 5 seconds to skip startup.nsh or any other key to continue.**
 Shell>

And i do not know how to press ESC, someone help!!!

**Configuration (required)**
* UTM Version: UTM SE first version
* OS Version: 17.4
* Device Model: Base Iphone 11 
* Is it jailbroken (name jailbreak used)? No
* How did you install UTM? App Store

**Comment by vernonstinebaker** [comment]
Pay attention to the screen preceding the screen that results in the Shell> 
You need to hit a key before getting to that screen to start the boot/install process. At least that's what worked for me. 
It took a very long time to install, including a couple of times it seemed to get frozen resulting in me restarting (after the initial installation was completed). On my M4 iPad, I can only say that it's sloooooooow. Really, really, slooooooow. (It does seem to be a little faster on second boot. Even slow, it's remarkable to see an x86 OS running as a VM on Apple Silicon. Thanks to the UTM contributors for making this happen.)

**Comment by matermel0n** [comment]
> Pay attention to the screen preceding the screen that results in the Shell> You need to hit a key before getting to that screen to start the boot/install process. At least that's what worked for me. It took a very long time to install, including a couple of times it seemed to get frozen resulting in me restarting (after the initial installation was completed). On my M4 iPad, I can only say that it's sloooooooow. Really, really, slooooooow. (It does seem to be a little faster on second boot. Even slow, it's remarkable to see an x86 OS running as a VM on Apple Silicon. Thanks to the UTM contributors for making this happen.)

but it didnt work?

**Comment by nguyenhuutuananh** [comment]
> UTM SE windows XP, just does not work. every time i try to boot it, **it says press ESC in 5 seconds to skip startup.nsh or any other key to continue.** Shell>
> 
> And i do not know how to press ESC, someone help!!!
> 
> **Configuration (required)**
> 
> * UTM Version: UTM SE first version
> * OS Version: 17.4
> * Device Model: Base Iphone 11
> * Is it jailbroken (name jailbreak used)? No
> * How did you install UTM? App Store

I met the same issue, just press any key to start the boot process as @vernonstinebaker said

**Reply by ntt77** [reply]
I press enter key many times but it still go to startup.nsh

**Comment by vernonstinebaker** [comment]
> but it didnt work?

Yes. It works.

**Comment by WindClan** [comment]
You accidentally enabled UEFI,
go into QEMU settings and disable it

**Reply by rxliuli** [reply]
Yes, I manually disabled it, and it took effect. But I don't remember manually enabling it...

**Comment by RedClaws1800** [comment]
@vernonstinebaker it really doesn't work bro

**Comment by RedClaws1800** [comment]
https://github.com/user-attachments/assets/a9bc2203-6f47-45c4-949a-2544136d8a9d
This was for Ubuntu on arm64 but the same exact problem occurs when trying windows XP
